asf_heat_pump_suitability/pipeline/prepare_features/garden_size.py

```diff
@@ -126,3 +127,28 @@ def generate_gdf_garden_size(
     )
 
     return gardens_gdf
+
+
+def deduplicate_df_garden_size(df: pl.DataFrame) -> pl.DataFrame:
```

Comment:
Just the docstring needs reviewing here. The code has already been reviewed as it was moved from a script into this function without changing.